src/gui/openfilemanager.cpp

```diff
@@ -103,27 +103,27 @@ void showInFileManager(const QString &localPath)
                 return;
 #endif
 
-        QString explorer = "explorer.exe "; // FIXME: we trust it's in PATH
-        QFileInfo fi(localPath);
-
-        // canonicalFilePath returns empty if the file does not exist
-        if (!fi.canonicalFilePath().isEmpty()) {
-            QString nativeArgs;
-            if (!fi.isDir()) {
-                nativeArgs += QLatin1String("/select,");
-            }
-            nativeArgs += QLatin1Char('"');
-            nativeArgs += QDir::toNativeSeparators(fi.canonicalFilePath());
-            nativeArgs += QLatin1Char('"');
-
-            QProcess p;
+            const QString explorer = "explorer.exe "; // FIXME: we trust it's in PATH
```

Comment:
indentation looks wrong in github as if there are 4 extra spaces

Reply:
I think the problem here is maybe a mix of tabs and spaces.

Reply:
no
I did check and I am seeing this when opening the file
![image](https://user-images.githubusercontent.com/168967/132306355-2afcee83-b491-459d-b57b-7a3db22280a7.png)

Reply:
`const QString explorer` has 4 extra spaces in from of it that should be removed

Reply:
@FlexW ping

Reply:
done